docs/designs/config.md

```diff
@@ -59,7 +59,7 @@
 
 ``` yml
 name: Azure.azure-documents
-default: msdocs.yml
+extend: msdocs.yml
```

Comment:
where do you think we put 'msdocs.yml'?

Reply:
It can be in a separate repo, restored in current config.